pyuvsim/source.py

```diff
@@ -116,16 +119,16 @@ def coherency_calc(self, time, telescope_location):
         return coherency_local
 
     @profile
-    def az_za_calc(self, time, telescope_location):
+    def alt_az_calc(self, time, telescope_location):
         """
-        calculate the azimuth & zenith angle for this source at a time & location
+        calculate the altitude & azimuth for this source at a time & location
 
         Args:
             time: astropy Time object
             telescope_location: astropy EarthLocation object
 
         Returns:
-            (azimuth, zenith_angle) in radians
+            (altitude, azimuth) in radians
```

Comment:
Looks like we switch between degrees and radians.  Should we try to be consistent?

Reply:
I think we've consistently used radians, although some comments reference degrees which I will change.

Reply:
we do use degrees in some testing, but I think that's ok as they're always converted to radians before calling the functions.

Reply:
yeah, thats what it looks like to me too.  Seems ok.